tools/gen_coverage_report.py

```diff
@@ -1,5 +1,3 @@
-#!/usr/bin/env python
```

Comment:
I think you don't need to remove these lines. They're harmless.

Reply:
If we are using `PYTHON_EXECUTABLE` to call them, these shebangs should be fine. If not, they still will be troubles, right? Why don't we just remove them to prevent future mistakes?